include/RocketMQClient.h

```diff
@@ -16,24 +16,7 @@
  */
 #ifndef __ROCKETMQCLIENT_H__
 #define __ROCKETMQCLIENT_H__
-
-#ifdef WIN32
-#ifdef ROCKETMQCLIENT_EXPORTS
-#ifdef _WINDLL
-#define ROCKETMQCLIENT_API __declspec(dllexport)
-#else
-#define ROCKETMQCLIENT_API
-#endif
-#else
-#ifdef ROCKETMQCLIENT_IMPORT
-#define ROCKETMQCLIENT_API __declspec(dllimport)
-#else
-#define ROCKETMQCLIENT_API
-#endif
-#endif
-#else
-#define ROCKETMQCLIENT_API
-#endif
+#include "CCommon.h"

```

Comment:
Keep the two sets of Interfaces independent of each other. This macro can be repeatedly defined here.
Please remove the reference of the c head file.

Reply:
the cpp also need exports class and function
1.CPP interface will depend on C interface .
2. C interface  independent the CPP interface

Reply:
This macro ROCKETMQCLIENT_API  should be definded in CPP and C head file independently.

Reply:
done,add a new file CPPCommon.h,pls check new commit.